catalog/commons/initramfs.yaml

```diff
@@ -0,0 +1,37 @@
+hooks:
+  generate_with_dracut:
+    remediate: dracut_initramfs
+    check: dracut
+    description: |
+      Create the initramfs with dracut.
+    actions:
+      - |-
+        EROOT=${EROOT:-/}
+        args="-N -f --xz -v"
+
+        # Extra Dracut modules included for features, like lvm and crypt
+        extra_mods="${DRACUT_ADD_MODULES}"
+
+        # Extra kernel modules included
+        extra_drivers="${DRACUT_ADD_DRIVERS}"
+
+        # Ensure KMODDIR is set
+        kmoddir="${KMODDIR:-/lib/modules/$(uname -r)}"
+
+        # Ensure that kver is set, like "debian-6.12.33_p1-mark"
+        kver="${KVER:-$(uname -r | cut -d '-' -f 1)}"
+
+        # Ensure that ktype is set, like "-debian"
+        ktype=${KTYPE:-$(uname -r | cut -d '-' -f 2)}
+
+        # Ensure that ksuffix is set, like "-mark"
+        ksuffix=${KSUFFIX:-$(uname -r | cut -d '-' -f 3)}
```

Comment:
As described in the previous comment probably you need to have `-debian-mark` in suffix because the suffix describes the kernel name under /lib/modules and just `mark` doesn't identify that is a debian kernel with MARK config and could be a problem if you have multiple kerneles with MARK configs.
I know that this is duplicate the debian string but the ktype is just a kernel sources identifier not used in the initrd image generation.